src/web-map.js

```diff
@@ -590,6 +590,32 @@ export class WebMap extends HTMLMapElement {
         this.dispatchEvent(new CustomEvent('zoomend', {detail:
           {target: this}}));
       }, this);
+    this.addEventListener('fullscreenchange', function(event) {
+      if (document.fullscreenElement === null) {
+        // full-screen mode has been exited
+        this._map.contextMenu.setViewFullScreenInnerHTML('view');
+      }else{
+        this._map.contextMenu.setViewFullScreenInnerHTML('exit');
+      }
+    });
+    this.addEventListener('keydown', function(event) {
+      // Check if Ctrl+R is pressed and map is focused
+      if (event.ctrlKey && event.keyCode === 82 && document.activeElement.nodeName === "DIV") {
```

Comment:
This looks a bit funny.  The event listener is checking to see that the active element is a div, which is very non-specific, so it will fire more often than you wish.